cfme/cloud/instance/openstack.py

```diff
@@ -54,6 +54,44 @@ def ui_powerstates_unavailable(self):
             'on': [self.START],
             'off': [self.SUSPEND, self.SOFT_REBOOT, self.HARD_REBOOT]}
 
+    def attach_volume(self, name, mountpoint=None):
+        initial_volume_count = self.volume_count
+        view = navigate_to(self, 'AttachVolume')
+        view.form.volume.select_by_visible_text(name)
+        if mountpoint:
+            view.form.mountpoint.fill(mountpoint)
+        view.form.submit_button.click()
+        view.flash.assert_success_message(
```

Comment:
Required: Flash messages assertion should not be present here, please assert it in tests

Reply:
How could I get view of volume attach on test in order to flash.assert_success_message

Reply:
ex. view = appliance.browser.create_view(navigator.get_class(instance_object, 'AttachVolume').VIEW)

Reply:
I tried above example and it keeps giving me  Available messages: []
Maybe i need to use "Details" instead or use:
 view = appliance.browser.create_view(BaseLoggedInPage)